bcda/cli.go

```diff
@@ -373,26 +381,34 @@ func sortCCLFFiles(cclfmap *map[string][]cclfFileMetadata, skipped *int) filepat
 		}
 		metadata, err := getCCLFFileMetadata(info.Name())
 		metadata.filePath = path
+		metadata.deliveryDate = info.ModTime()
```

Comment:
I think we need to be checking this in the existing tests.  Maybe even touch a file and see if it changes, open it and see that it doesn't?